src/core/thread/mle.cpp

```diff
@@ -141,10 +142,12 @@ Mle::Mle(Instance &aInstance)
 
     mMeshLocal64.InitAsThreadOriginRealmLocalScope();
     mMeshLocal64.GetAddress().GetIid().GenerateRandom();
+    mMeshLocal64.mPreferred = false;
```

Comment:
Is it conceptually incorect to set ML64 as `not preferred`? ML64 is preferred within the Thread network, but may not be preferred on host interface. I think we need to differentiate the two cases.